javascript/node/selenium-webdriver/test/window_test.js

```diff
@@ -99,6 +99,12 @@ test.suite(function(env) {
     return driver.wait(forPositionToBe(x, y), 1000);
   });
 
+  it('can open a new window', async function() {
+    await driver.switchTo().newWindow()
+
+    assert.equal((await driver.getAllWindowHandles()).length, 2)
+  })
```

Comment:
... and here we could check that we're in the new windows, that means the new window handle is not equal to the original one.